pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "linerate"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
I think we should use 2.0.0 since we are implementing a breaking change

Reply:
It was already increased a major version (from 0.0.7) in a previous PR, which will also be part of this release. Reflecting on that, 1.0.0 would make more sense than 1.0.1.